sale_automatic_workflow/models/automatic_workflow_job.py

```diff
@@ -74,12 +69,14 @@ def _create_invoices(self, create_filter):
 
     def _do_validate_invoice(self, invoice):
         """Validate an invoice"""
-        with savepoint(self.env.cr), force_company(self.env,
-                                                   invoice.company_id):
-            # FIX Why is this needed for certain invoices
-            # in enterprise in multicompany?
-            invoice.with_context(
-                force_company=invoice.company_id.id).action_invoice_open()
+        with savepoint(self.env.cr):
+            try:
+                invoice.with_context(
+                    force_company=invoice.company_id.id
+                ).action_invoice_open()
+            except Exception:
+                _logger.exception('Error on validating invoice {}'.format(invoice.id))
+                raise Exception
```

Comment:
@i-vyshnevska to reraise something without shadowing the exception, you should just use `raise`.
```suggestion
                raise
```